internal/network/command_list.go

```diff
@@ -0,0 +1,68 @@
+package network
+
+import (
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/output"
+)
+
+func (c *command) newListCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "Display networks in the current environment.",
+		Args:  cobra.NoArgs,
+		RunE:  c.list,
+	}
+
+	pcmd.AddContextFlag(cmd, c.CLICommand)
+	pcmd.AddEnvironmentFlag(cmd, c.AuthenticatedCLICommand)
+	pcmd.AddOutputFlag(cmd)
+
+	return cmd
+}
+
+func (c *command) list(cmd *cobra.Command, _ []string) error {
+	networks, err := c.getNetworks()
+	if err != nil {
+		return err
+	}
+
+	list := output.NewList(cmd)
+	for _, network := range networks {
+		zones := network.Spec.GetZones()
```

Comment:
add a nil pointer check for `network.Spec` and `network.Status` to handle odd responses

Reply:
added the checks